packages/server/postgres/queries/getAtlassianAuthByUserIdTeamId.ts

```diff
@@ -0,0 +1,56 @@
+import getPg from '../getPg'
+import {
+  getAtlassianAuthByUserIdTeamIdQuery,
+  IGetAtlassianAuthByUserIdTeamIdQueryResult
+} from './generated/getAtlassianAuthByUserIdTeamIdQuery'
+
+interface AtlassianAuth
+  extends Omit<IGetAtlassianAuthByUserIdTeamIdQueryResult, 'jiraSearchQueries'> {
+  jiraSearchQueries: {
+    id: string
+    queryString: string
+    projectKeyFilters?: string[]
+    lastUsedAt?: Date
+    isJQL: boolean
+  }[]
+}
+
+const getAtlassianAuthByUserIdTeamId = async (userId: string, teamId: string) => {
+  const [
+    {
+      accessToken,
+      refreshToken,
+      createdAt,
+      updatedAt,
+      isActive,
+      jiraSearchQueries,
+      cloudIds,
+      scope,
+      accountId
+    }
+  ] = await getAtlassianAuthByUserIdTeamIdQuery.run({teamId, userId}, getPg())
+
+  return {
+    accessToken,
+    refreshToken,
+    createdAt,
+    updatedAt,
+    isActive,
+    cloudIds,
+    scope,
+    accountId,
+    teamId,
+    userId,
+    jiraSearchQueries: jiraSearchQueries.map((jsq: any) => {
+      return {
+        id: jsq.id,
+        queryString: jsq.queryString,
+        projectKeyFilters: jsq.projectKeyFilters,
+        lastUsedAt: jsq.lastUsedAt ? new Date(jsq.lastUsedAt) : undefined,
+        isJQL: jsq.isJQL
+      }
+    })
+  } as AtlassianAuth
```

Comment:
@mattkrick I know it's a bit verbose but it allows us to shape the object exactly as we want (maybe validate in the future all the props?). Please let me know if that's acceptable or if there's some cleaner solution 🙏